examples/evaluation/huggingface-evaluation.ipynb

```diff
@@ -0,0 +1,1340 @@
+      "    )\n"
+     ]
+    }
+   ],
+   "source": [
+    "from mlflow.metrics.genai import answer_correctness\n",
+    "\n",
+    "answer_correctness_metric = answer_correctness(model=\"openai:/gpt-4\")\n",
+    "\n",
+    "print(answer_correctness_metric)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "ac9cdf24-d024-4543-a292-371bb301041f",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Evaluate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We need to set our OpenAI API key, since we are using GPT-4 for our LLM-judged metrics."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 47,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "3b56fb6b-2272-4ddf-b297-b222a13efb77",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ[\"OPENAI_API_KEY\"] = \"redacted\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Call `mlflow.evaluate()` on the first 10 rows of the data. Using the 'text-summarization' model, we get toxicity, readability metrics, and rouge score as builtin metrics. We also pass in the two metrics we defined above into the extra_metrics parameter to be evaluated."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 48,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "da8e792d-7750-40c2-b55f-4f902e6d9442",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Downloading artifacts:   0%|          | 0/13 [00:00<?, ?it/s]2023/12/01 15:23:29 INFO mlflow.store.artifact.artifact_repo: The progress bar can be disabled by setting the environment variable MLFLOW_ENABLE_ARTIFACTS_PROGRESS_BAR to false\n",
+      "Downloading artifacts: 100%|██████████| 13/13 [00:07<00:00,  1.68it/s] \n",
+      "2023/12/01 15:23:36 WARNING mlflow.transformers: Could not specify device parameter for this pipeline type\n",
+      "2023/12/01 15:23:37 INFO mlflow.models.evaluation.base: Evaluating the model with the default evaluator.\n",
+      "2023/12/01 15:23:37 INFO mlflow.models.evaluation.default_evaluator: Computing model predictions.\n",
+      "Token indices sequence length is longer than the specified maximum sequence length for this model (789 > 512). Running this sequence through the model will result in indexing errors\n",
```

Comment:
This model isn't big enough for this task, right? Can a model be selected that won't have generated encoding errors?